src/components/Social.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class Social extends React.Component {
+  render() {
+    return (
+      <a href={ this.props.href } title={ this.props.title }><i className={ this.props.icon }></i></a>
```

Comment:
1. I'd like an <Icon /> component.
2. I'd like this <Icon /> component to be passed as a  `PropTypes.element.isRequired` to the relevant components